compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerToAIE.cpp

```diff
@@ -396,9 +384,6 @@ LogicalResult coreToAIE(IRRewriter &rewriter, AMDAIE::CoreOp coreOp,
               return coreReleaseOpToAIE(rewriter, releaseOp, mapper,
                                         toBeErased);
             })
-            .Case<linalg::LinalgOp>([&](auto linalgOp) {
```

Comment:
Just reading the description and PR, this seems like it is dropping general handling of `LinalgOp`. Is that to mean that we dont expect `LinalgOp`s to survive till here?

Reply:
Instead of handling them as a special case: https://github.com/nod-ai/iree-amd-aie/blob/57cd0bca89917e47639b9189683ac71b5d399f3c/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerToAIE.cpp#L399

They are now handled like other not-special ops: https://github.com/nod-ai/iree-amd-aie/blob/57cd0bca89917e47639b9189683ac71b5d399f3c/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerToAIE.cpp#L411

We (I) do expect LinalgOps to survive til here.